packages/schematics/src/command-line/deps-calculator.spec.ts

```diff
@@ -0,0 +1,229 @@
+import { DepsCalculator, DependencyType, Deps } from './deps-calculator';
```

Comment:
I'd have the following tests here:

1. should add a new edge incrementally.
2. should remove an edge incrementally.
3. should add a node incrementally.
4. should remove a node and its edges incrementally.
5. should rename a node (probably remove+add, but a test is still useful)
6. should change an edge's type